packages/ui-components/src/components/DataTable/useDataTableExport.js

```diff
@@ -28,9 +32,21 @@ export const useDataTableExport = (columns, data, title, startDate, endDate) =>
         ? tableData.map(row =>
             tableColumns.map(col => {
               const value = row.values[col.id];
-              const valueIsPercentageString = value.includes('%');
-              const num = valueIsPercentageString ? value : parseFloat(value);
```

Comment:
Going back in time a little. What happens if we just swap this `parseFloat(value)` to `Number(value)`?

Reply:
So I think if you do `Number(true)` then it becomes `1` but if you do `parseFloat(true)` then it becomes `NaN`, which is what we want I think...

Reply:
That's true. We'd want to avoid specifically cases where the value is a boolean or an empty string. But it'd reject percentage strings as well as almost all our date formats (except `YYYY`, which I think we can just let parse to number). Am I missing any other edge cases?

```
const num = (value && typeof value !== 'boolean') ? Number(value) : value;
```

Reply:
So I have a check for `typeof value === 'string' and isNaN(value)` which will catch any strings that aren't considered stringified numbers, e.g. percentages and date strings. And then it handles dates within that, but converting to a moment object, and if it is a date that is valid and is a 'D*' format (ie.g. anything with a day in it) then it converts to 'DD/MM/YYYY'.
Any other values (e.g. booleans, strings that are stringified numbers, and numbers) go to the parseFloat. So yeah, we could do that as well at this same stage. But in order to format dates appropriately then we need to do the rest of the steps above as well, unless Regina/Georgia make the decision that not all dates need to be converted, and that as long as they don't parse to numbers then that's fine (I have asked on the ticket)

Reply:
Yeah, looking at the code you've written it looks like it'd work. It just feels like a bit of code smell.

Personally I'd push back that we shouldn't be reformatting the dates at all. It's a raw data export. It should be the raw data formatting.

Reply:
Yeah makes sense - let me wait on the answer on whether we can keep the formatting as is and go from there